CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+#### 0.27.0-dev
```

Comment:
If you plan on releasing this now, then bump the pubspec version as well

Reply:
I'm actually planning to do a trick here and release v0.26.1 since all changes since 0.26.0 are backwards-compatible. Keeping this as-is though since the current pubspec version is 0.27.0-dev and I've got one more update to land since we set the dev version. 

Typically we update this value in a release commit that updates the pubspec, the readme and the changelog atomically.